src/main/java/culturemedia/controllers/CultureMediaController.java

```diff
@@ -0,0 +1,26 @@
+package culturemedia.controllers;
+
+import java.util.*;
+
+import culturemedia.exception.VideoNotFoundException;
+import culturemedia.model.Video;
+import culturemedia.service.impl.CultureMediaServiceImpl;
+
+public class CultureMediaController {
+
+    private final CultureMediaServiceImpl cultureMediaService;
+
+
+    public CultureMediaController(CultureMediaServiceImpl cultureMediaService) {
+        this.cultureMediaService = cultureMediaService;
+    }
+
+
+    public List<Video> find_allVideos() throws VideoNotFoundException {
+        List<Video> videos = null;
```

Comment:
On the line where you declare List<Video> video equal to null is not that useful, since the next line says it equals cultureMediaService.findAll(), so we could save line 21 of code, by putting cultureMediaService.findAll () instead of Null on line 20